scripts/deduplicate.py

```diff
@@ -0,0 +1,90 @@
+import argparse
```

Comment:
Can you add more information on what is the intended use of this script? 
Deduplication is certainly useful, but newcomers might not be so familiar